### ISSUE TITLE
<View fixed> is not rendered properly

### ISSUE THREAD

**Issue by PerfectPixel** [diegomura/react-pdf#253]
**OS:**
macOS High Sierra 10.13.5

**React-pdf version:**
0.7.6
1.0.0

**Description:**
_0.7.6_
`<View fixed>...</View>` does not have any effect. The element is only visual on the first page.
_1.0.0_
`<View fixed>...</View>` is displayed on every page, however, on every page the fixed content is indented further.

1st page
![screen shot 2018-07-03 at 13 37 50](https://user-images.githubusercontent.com/2388847/42217670-6063f652-7ec6-11e8-995e-37764e74756b.png)

2nd page
![screen shot 2018-07-03 at 13 37 54](https://user-images.githubusercontent.com/2388847/42217674-62803004-7ec6-11e8-8406-ec49e38c0436.png)

3rd page
![screen shot 2018-07-03 at 13 37 57](https://user-images.githubusercontent.com/2388847/42217676-63bfb16a-7ec6-11e8-9cb0-3c83e31bbf38.png)

4th page
![screen shot 2018-07-03 at 13 38 00](https://user-images.githubusercontent.com/2388847/42217678-651567e4-7ec6-11e8-919e-2beadf902e3c.png)

**How to replicate issue including code snippet (if applies):**
```
...
  <Document>
    <Page style={styles.body} wrap>
      <View fixed>
        <Text style={styles.header}>
          ~ Created with react-pdf ~
        </Text>
      </View>
      ...
    </Page>
  </Document>
...
```

**Comment by aslattery** [comment]
Experiencing the same on `1.0.0-alpha.8`.

**Comment by dberringer** [comment]
I'm seeing the same thing on `1.0.0-alph.15`

**Comment by diegomura** [comment]
Probably going to be fixed in #285. Thanks for reporting it
